src/Containers/LandingPage/NavBar/navbar/DesktopNavMenu/DesktopNavMenu.js

```diff
@@ -0,0 +1,29 @@
+import classes from './DesktopNavMenu.module.css'
+import {NavLink} from 'react-router-dom'
+
+const DesktopNavMenu=()=>{
+    
+
+    return(
+        <div className={`${classes["navigation-items-desktop"]} ${classes["sticky-status-margin-top"]}`}>
+                <p className={classes["nav-button"]}>
+                    <NavLink to="/" exact activeClassName={classes["active-desktop"]}>
+                        Home
+                    </NavLink>
+                </p>
+                <p className={classes["nav-button"]}>
+                    <NavLink to="/my-playlist" exact activeClassName={classes["active-desktop"]}>
+                        My Playlists
+                    </NavLink>
+                </p>
+                <p className={classes["nav-button"]}>
+                    <NavLink to="/history" activeClassName={classes["active-desktop"]}>
+                        History
+                    </NavLink>
+                </p>
+                
+            </div>
+    )
+}
+
```

Comment:
There is a white space below this component and above the home page image. You can fix this by checking margin-bottom and margin-top or any other way.

Reply:
Good catch! I'll remove the bottom margin